MAUI/Barcode-Generator/one-dimensional-symbology.md

```diff
@@ -0,0 +1,450 @@
+    barcode.ShowText = true;
+    this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Code93 Symbology](images/one-dimensional-symbology/maui-code93-symbology.png)
+
+## Code128
+
+`Code 128` is a highly efficient, high-density linear barcode symbology that allows the encoding of alphanumeric data. It is capable of encoding full ASCII character set and extended character sets. This symbology contains the checksum digit for verification and the barcode can also be verified character-by-character for the parity of each data byte.
+
+The `Code128` symbology encodes the input symbols supported by `Code128A`, `Code128B`, `Code128C`. The default symbology type of barcode generator is `Code128`.
+
+{% tabs %}
+
+{% highlight xaml hl_lines="5" %}
+
+    <barcode:SfBarcodeGenerator Value="CODE128"
+                                ShowText="True"
+                                HeightRequest="150">
+        <barcode:SfBarcodeGenerator.Symbology>
+                <barcode:Code128 Module="2" />
+        </barcode:SfBarcodeGenerator.Symbology>
+    </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="4" %}
+
+    SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+    barcode.HeightRequest = 150;
+    barcode.Value = "CODE128";
+    barcode.Symbology = new Code128() { Module = 2 };
+    barcode.ShowText = true;
+    this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Code128 Symbology](images/one-dimensional-symbology/maui-code128-symbology.png)
+
+## Code128A
+
+`Code128A` (or Chars Set A) includes all the standard upper case U.S. alphanumeric keyboard characters and punctuation characters together with the control characters, (characters with ASCII values from 0 to 95 inclusive), and seven special characters.
+
+{% tabs %}
+
+{% highlight xaml hl_lines="5" %}
+
+    <barcode:SfBarcodeGenerator Value="CODE128A"
+                                ShowText="True"
+                                HeightRequest="150">
+        <barcode:SfBarcodeGenerator.Symbology>
+            <barcode:Code128A Module="2" />
+        </barcode:SfBarcodeGenerator.Symbology>
+    </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="4" %}
+
+    SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+    barcode.HeightRequest = 150;
+    barcode.Value = "CODE128A";
+    barcode.Symbology = new Code128A();
+    barcode.ShowText = true;
+    this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Code128A Symbology](images/one-dimensional-symbology/maui-code128a-symbology.png)
+
+## Code128B
+
+`Code128B` (or Chars Set B) includes all the standard upper case alphanumeric keyboard characters and punctuation characters together with lower case alphabetic characters (characters with ASCII values from 32 to 127 inclusive), and seven special characters.
```

Comment:
`Code128B` (or Chars Set B) includes all the standard upper case alphanumeric keyboard characters and punctuation characters, as well as the lower case alphabetic characters (characters with ASCII values from 32 to 127 inclusive) and seven special characters.

Reply:
changes updated.